src/pushsource/_impl/model/vms.py

```diff
@@ -79,6 +90,13 @@ class VMIPushItem(PushItem):
     boot_mode = attr.ib(type=BootMode, default=None, validator=optional(in_(BootMode)))
     """Boot mode supported by the image (if known): uefi, legacy, or hybrid (uefi + legacy)."""
 
+    cloud_info = attr.ib(
+        type=VMICloudInformation,
+        default=None,
+        validator=optional((instance_of(VMICloudInformation))),
+    )
+    """Cloud provider information of its short name and account alias."""
```

Comment:
I would prefer that this is worded in such a way that it doesn't imply these are the *only* attributes.

For example, instead of "of its" it could be something like:

```suggestion
    """Cloud provider information, such as the provider's short name and account alias."""
```

"such as" communicating that these are only examples and you should check the class docs if you want to know everything.

The reason for this is that, if someone later needs to add more attributes to VMICloudInformation, I think it's likely they will not come back and update this doc string.  The doc string therefore would be best worded with future additions in mind to avoid becoming inaccurate.

Reply:
Fixed, thanks a lot!